pkg/blockchain/plugin.go

```diff
@@ -85,7 +88,7 @@ type Callbacks interface {
 	// submitted by us, or by any other authorized party in the network.
 	//
 	// Error should will only be returned in shutdown scenarios
-	BatchPinComplete(batch *BatchPin, signingIdentity string) error
+	BatchPinComplete(batch *BatchPin, signingKey *fftypes.VerifierRef) error
```

Comment:
A bit of faff throughout this PR in dealing with the fact that we don't just have a string for a verifier any more, you have to describe its type. This is necessary to allow a single DB table, but as we have separate types for Fabric/Ethereum we have to pass it around a bit.